test/Database/LSMTree/Class/Monoidal.hs

```diff
@@ -217,7 +217,7 @@ withTableOpen :: forall h m k v a.
   -> SnapshotName
   -> (h m k v -> m a)
   -> m a
-withTableOpen sesh snap = bracket (open sesh snap) close
```

Comment:
We could also rename this bit to `withTableOpenSnapshot` or some similar name

Reply:
I called it `withTableFromSnapshot`